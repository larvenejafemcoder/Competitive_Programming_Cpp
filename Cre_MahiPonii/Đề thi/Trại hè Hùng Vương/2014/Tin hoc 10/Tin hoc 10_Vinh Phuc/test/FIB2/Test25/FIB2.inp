42
39 43964366
28 331254
20 6275
21 10842
35 12774215
26 88418
42 398067277
3 1
2 1
9 27
29 362240
18 2772
32 2424559
45 1155802482
32 1645282
24 55972
4 4
12 1
26 52979
32 2513426
15 913
1 1
20 10188
11 10
23 40101
10 44
33 779836
5 6
34 661901
26 183091
16 172
14 600
2 1
23 13856
26 169055
37 3128377
4 5
26 168285
43 473690116
22 25754
3 1
40 77417573
