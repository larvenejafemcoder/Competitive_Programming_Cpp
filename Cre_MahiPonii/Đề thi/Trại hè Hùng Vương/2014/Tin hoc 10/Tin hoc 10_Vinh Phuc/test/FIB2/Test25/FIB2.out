16792894
126528
2397
4141
4879316
33773
152048170
0
1
10
138363
1059
926099
441477264
628442
21380
2
1
20236
960043
349
0
3892
4
15317
17
297871
2
252824
69935
66
229
1
5293
64573
1194934
2
64279
180933524
9837
0
29570882
